recipe/meta.yaml

```diff
@@ -32,6 +32,7 @@ requirements:
     - pymdown-extensions >=6.3
   run_constrained:
     - mkdocstrings-python-legacy >=0.2
+    - mkdocstrings-python >=5.2
```

Comment:
What do you think about having `mkdocstrings-python-legacy` and `mkdocstrings-python` as hard deps (once the conda feedstock has finally been created ofc)?

Pretty sure we will have more usecase where people want to have handlers than the opposite. That would also keep the previous behaviour where all the deps are within `mkdocstring`.

ping @bandersen23

Reply:
That's a good idea, I'll update it in #24 when the feedstocks are all officially created.

Reply:
Thanks.

Reply:
Done in ba88b67 thru #24.